partner_contact_nationality/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    nationality_id = fields.Many2one("res.country", "Nationality")
```

Comment:
Strictly speaking this is not correct. There are nationalities spread over multiple states, and states containing multiple nationalities. Not blocking.